test/runtests.jl

```diff
@@ -1,12 +1,58 @@
 using Test
-using LinearAlgebra, Statistics, TopoPlots, CairoMakie
+using LinearAlgebra, Statistics, TopoPlots, CairoMakie, FileIO
+using PyCall
+try
+    PyCall.pyimport("mne")
+catch e
+    # Hmpf, how else can we make sure a test dependency exists?
+    # I tried adding Conda for PyPlot, which then installs matplotlib automatically.
+    # It looks like this messed with mne, so one of those will need to have an ugly install?!
+    run(PyCall.python_cmd(`-m pip install mne`))
```

Comment:
The build step of PyMNE 0.1.x should do this for you

Reply:
Well it doesn't somehow, since I added Conda to the test deps...

Reply:
Conda is changing which Python is used after PyMNE has already been built.